packages/store/src/store.js

```diff
@@ -3,6 +3,7 @@
 // @ts-check
 
 import { assert, details as X, q } from '@agoric/assert';
+import { isEmptyNonRemotableObject } from './helper';
```

Comment:
Once there is a helper file it tends to accumulate more helpers, so better to give it a plural name from the beginning.